packages/firebase_ml_vision/ios/Classes/ImageLabeler.m

```diff
@@ -0,0 +1,57 @@
+#import "FirebaseMlVisionPlugin.h"
```

Comment:
This file was renamed to `ImageLabeler.m` changes can be seen [here](https://github.com/flutter/plugins/pull/1364/commits/c9c42d8175dd30f21243d926165af466e887f590).

Reply:
I would probably capitalize the L in MLVision